AwesomeMedia/Classes/Extensions/AVPlayerItemExtensions.swift

```diff
@@ -100,22 +100,24 @@ extension AVPlayerItem {
     
     public var isVideo: Bool {
         for playerItemTrack in tracks {
-            if playerItemTrack.assetTrack.hasMediaCharacteristic(
-                AVMediaCharacteristic.visual) {
-                return true
+            if let assetTrack = playerItemTrack.assetTrack {
+                if assetTrack.hasMediaCharacteristic(
```

Comment:
This `if let` and the `if` inside, is one dependent on the another, then we can do all in one only line. `if let, condition`. I've changed it, for both updates, then you can see better how it looks like @maail.

Reply:
Cool nice. Missed that. Thanks